src/page-multi-facility/FacilityInputForm.tsx

```diff
@@ -178,7 +187,26 @@ const FacilityInputForm: React.FC<Props> = ({ scenarioId }) => {
   const closeDeleteModal = () => {
     updateShowDeleteModal(false);
   };
-  const popupItems = [{ name: "Delete", onClick: openDeleteModal }];
+  const onDuplicateFacility = async () => {
+    if (facility) {
+      await rejectionToast(
```

Comment:
Thank you for remembering to take care of the shared scenario case 🙏

Reply:
TBH I'm didn't know I had taken care of it 😅  could you explain how it is taken care of? I'm not very familiar with that new feature yet.

Reply:
😄 

By including this `rejectionToast` call, a user can only modify Scenarios and Facilities they own.  So if you're viewing a facility that lives within a Scenario owned by someone else and you try to duplicate that facility, an error toast will be shown.  This is the expected and desired behavior because we don't want people duplicating facilities within someone elses's scenario.